.github/workflows/run-test-manual.yml

```diff
@@ -142,6 +142,8 @@ jobs:
 
       - name: Merge playwright reports
         uses: ./.github/actions/merge-pw-reports
+        with:
+           PASSWORD_FOR_DECODING_ARTIFACT: ${{ secrets.E2E_USER_PASSWORD }}
```

Comment:
are you sure to use exact same password as for user login? isn't better to have separate variable for it?

Reply:
I think if anyone had our user password, separate password for tests results won't do much good

Reply:
the password can stay the same, the variable can point to it, I'm saying that we share the same variable for 2 different things

Reply:
Changed